src/app/randomThings.js

```diff
@@ -0,0 +1,27 @@
+export const randomInteger = (min, max) => ( // Inclusive.
+  Math.floor(Math.random() * (max - min + 1)) + min
+)
+
+export const randomRange = (min, max) => {
+  const x = randomInteger(min, max)
+  const y = randomInteger(min, max)
+
+  return x === y
+    ? randomRange(min, max)
```

Comment:
This doesn't guarantee a range. What about something like this:
```js
const randomRange = (rangeBoundary1, rangeBoundary2) => {
	if (rangeBoundary1 === rangeBoundary2) {
  	throw new Error ('Pass two different values')
  }
  
	const v1 = randomInteger(rangeBoundary1, rangeBoundary2)
        const rangeArray = [v1]
  
  while (returnValue.length < 2) {
  	const v2 = randomInteger(rangeBoundary1, rangeBoundary2)
    if (v2 > v1) returnValue.push(v2)
    if (v2 < v1) returnValue.unshift(v2)
    continue
  }
  
  return rangeArray
}
```

Reply:
I like the error checking. I added a note in src/app/initialSettings and now check in both `randomInteger()` and `randomRange()` that `max` and `min` are integers and `max` > `min`.

As for _this doesn't guarantee a range..._ it does! If the random results generate from `min` and `max` are equal, `randomRange()` calls itself again for new random numbers until it gets differing values. Or am I misunderstanding?

Reply:
For the record: as discussed, your version is a lot more efficient in that in case of matching results I replace both random numbers and re-sort, but I have an emotional attachment to the simplicity of the existing code. 🤦‍♂️ 🙄 🤷‍♂️